hello-world-notification-microservice/src/main/java/c8y/example/notification/helloworld/NotificationExample.java

```diff
@@ -56,6 +58,11 @@ public void onSubscriptionAdded(MicroserviceSubscriptionAddedEvent event) throws
         runExample();
     }
 
+    @EventListener
+    public void destroy(MicroserviceSubscriptionRemovedEvent event) throws Exception {
+        this.client.close();
```

Comment:
shouldn't we check `if client != null`?

Reply:
Done and renamed method to be more symmetric or related to the onSubscriptionAdded(...) lifecycle method